spec/deaf_grandma_spec.rb

```diff
@@ -7,12 +7,18 @@
   it "says 'SPEAK UP SONNY!' when we speak regularly" do
     expect(script.speak("Hi Grandma")).to eq "SPEAK UP SONNY!"
   end
-  
+
   it "says 'NOT SINCE 1964!' when we yell" do
     #implement your test here
+      expect(script.speak("YELL")).to eq "NOT SINCE 1964!"
   end
 
   it "EXTRA CREDIT: How would you test yelling BYE?" do
     #implement your test here
+      expect(script.speak("BYE")).to eq "NOT SINCE 1964!"
+      expect(script.speak("BYE")).to eq "NOT SINCE 1964!"
+    #expect(script.speak("BYE")).to eq "SEE YOU LATER SONNY!"
+      expect { script.speak("BYE") }.to raise_error(SystemExit)
```

Comment:
cool.  Seems like you'd also want to test the "SEE YOU LATER SONNY" part, though.